modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -94,6 +95,19 @@ public BaseSectionDisplayContext(
 			httpServletRequest.getAttribute(InfoDisplayWebKeys.INFO_ITEM));
 	}
 
+	public Map<String, Object> getAdditionalProps() {
+		return HashMapBuilder.<String, Object>put(
+			"autocompleteURL",
+			() -> StringBundler.concat(
+				"/o/search/v1.0/search?emptySearch=",
+				"true&entryClassNames=com.liferay.portal.kernel.model.User,",
+				"com.liferay.portal.kernel.model.UserGroup&nestedFields=",
+				"embedded")
+		).put(
+			"collaboratorsURLs", () -> _getCollaboratorsURLs()
```

Comment:
`collaboratorsURLs` sound a bit off. Can you rename it to `collaboratorURLs`? (first word in singular).